README.md

```diff
@@ -161,6 +161,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     Sponsored dinners in San Francisco where the women coming from all over are invited to share their experiences, alongside prominent speakers from the female tech communits. The events also offer networking and potential recruiting by the sponsoring company. It has been hosted by 100+ companies and has several events per month.
 
+  - [Fortune Most Powerful Women Summit](http://www.fortuneconferences.com/most-powerful-women-summit-2016/)
+
+    The Women in Tech Summit inspires, educates and connects women in the technology industry. Join us for a unique combination of deep-dive, hands-on tech workshops; information and discussions about careers in tech and how to pursue them; and connection and networking opportunities with other women in various aspects and careers in technology.
```

Comment:
Remove the 'Join us for a unique combination bit', otherwise, it looks okay. Remember that this is not marketing, rather just informative. Do that and I'll merge.

Thanks for your contribution.